ci/raw_init.sh

```diff
@@ -3,5 +3,6 @@
 set -ex
 
 sh ./rustup-init.sh --default-toolchain none -y
+# shellcheck source=/dev/null
```

Comment:
similar to [ci/cirrus-templates/script.bash](https://github.com/rust-lang/rustup/pull/3121/files#diff-53a66607b98fd4e58616b06387fd72e3fc80f3874086f3bd3376878792e1833d) this should be working.